src/idl_gen_kotlin.cpp

```diff
@@ -1334,7 +1356,8 @@ class KotlinGenerator : public BaseGenerator {
                           const std::string &params,
                           const std::string &returnType,
                           const std::function<void()> &body,
-                          bool gen_jvmstatic = false) {
+                          bool gen_jvmstatic = false,
+                          bool gen_experimental_unsigned_types = false) {
```

Comment:
Why do functions need to be aware of the annotation? Isn't the class itself annotated not enough?